README.adoc

```diff
@@ -2537,18 +2544,29 @@ class Person
 end
 ----
 
-==== Non-declarative Methods That Have "keyword" Status in Ruby [[method-invocation-parens-non-declarative-keyword]]
+===== Non-Declarative Methods That Have "keyword" Status in Ruby [[method-invocation-parens-non-declarative-keyword]]
+
+There are two schools of thought within the community: one says to always omit parentheses for non-declarative methods that have "keyword" status (e.g., `Kernel` instance methods), and the other says not to omit them.
 
-Can omit parentheses for methods that have "keyword" status in Ruby, but are not declarative:
+The omit-parentheses school says that certain non-declarative methods feel like they have "keyword" status.
+Since they feel like keywords, the code reads better *without* parentheses because it looks like keywords rather than method calls.
+
+The use-parentheses school says that there's nothing special about these non-declarative methods, their semantics are ordinary, and which methods have "keyword" status is not well-defined.
+Therefore, *with* parentheses, it's easier to write with a consistent style since you don't have to worry about defining which non-declarative methods have "keyword" status, and the code reads better because it's just ordinary method calls.
+
+There are respected Rubyists from both schools of thought within the community, so choose the one you identify with most and be consistent.
```

Comment:
As this can be repeated about several guidelines I'd rather skip it.

Reply:
OK, I force-pushed an amended commit that removes that.